app/models/group/sektions_mitglieder.rb

```diff
@@ -16,7 +16,10 @@ class Mitglied < ::Role
 
     validates :end_on, presence: true
 
```

Comment:
Hier fehlt noch entweder ein `attribute :family_id` oder eine Migration die eine neue DB Spalte einfügt. ist aber gar nicht so genau gespez'd

Reply:
Danke nils, ich dachte das gäbs schon, hab die migration ergänzt und in der spec noch ein reload gemacht